Azure/communication-ui-library-ios
# [Feature] Introduce the LobbyOverlayViewModel with localization

## Purpose
Introduce new view model to include localization provider. 
Using this to have dynamic localized string in lobby overlay

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Using the new settings page to change the language to something other than `en`
Join teams meeting 
Check new content in the lobby overlay.
